File: c-exercicios/lista01/Eleva_X_A_Um_Y_Positivo.c
/**********************************************/
/**** Este programa calcula o resultado de ****/
/**** um número x elevado a um número y    ****/
/**** inteiro positivo.                    ****/
/**********************************************/

/***********************************/
/**** Autor: Aron Daniel Lopes  ****/
/**** Data: 12/04/2011			****/
/***********************************/

#include<stdio.h>

//%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%//

int y, n;
float x, resultado;

//%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%//

//===================================================================//

main()
{
	printf("Este programa calcula o valor de um numero x elevado a um numero y inteiro positivo.\n");
	printf("Digite o numero x: \n");
	scanf("%f" , &x);
	printf("Digite o numero y: \n");
	scanf("%d" , &y);
	if(y>0) //Verifica se y é positivo.
	{
		resultado=1;
		for(n=1;n<=y;n++) //Laço para elevar o número.
		{
			resultado=resultado*x;
		}
		printf("Resultado: %f\n", resultado);
	}
	else
	{
		printf("Valor de y nao e positivo. Entrada invalida.\n"); 
	}
	return 0;
}

//===================================================================//

File: python-exercicios/lista01/Eleva_X_A_Um_Y_Positivo.py
#**********************************************/
#**** Este programa calcula o resultado de ****/
#**** um número x elevado a um número y    ****/
#**** inteiro positivo.                    ****/
#**********************************************}

#***************************************************************/
#**** Autor: Aron Daniel Lopes e Marco Aurélio Dilda Campos ****/
#**** Data: 12/04/2011			                            ****/
#***************************************************************}

#{/===================================================================/}

print "Este programa calcula o valor de um numero x elevado a um numero y inteiro positivo."
x= input("Digite o numero x: ")
y= input("Digite o numero y: ")
if(y>0): #{ Verifica se y é positivo. }
	resultado = 1.0
	for n in range(1, y+1): #(* Laço para elevar o número. *)
			resultado=resultado*x
	print "Resultado: " +str(resultado)
else:
	print "Valor de y nao e positivo. Entrada invalida."

#//===================================================================//